wolfcrypt/src/ecc.c

```diff
@@ -8204,12 +8196,12 @@ int ecc_mul2add(ecc_point* A, mp_int* kA,
   /* allocate memory */
   tA = (unsigned char*)XMALLOC(ECC_BUFSIZE, heap, DYNAMIC_TYPE_ECC_BUFFER);
   if (tA == NULL) {
-     return GEN_MEM_ERR;
+     return MP_MEM;
```

Comment:
This should be MEMORY_E.

Reply:
These `malloc` failures all return `MP_MEM` per the incumbent code.  It's safer not to make unnecessary functional changes in these large refactor campaigns -- the error codes in particular tend to be fidgity.